tools/Common/IceRpc.BuildTelemetry/TelemetryTask.cs

```diff
@@ -23,9 +23,24 @@ public class TelemetryTask : ToolTask
     public string Idl { get; set; } = "";
 
     /// <summary>
-    /// Gets or sets the if the compilation contained any Slice1 files.
+    /// Gets or sets a value indicating whether the compilation contained any Slice1 files.
     /// </summary>
-    public string? ContainsSlice1 { get; set; } = "";
+    public bool ContainsSlice1 { get; set; } = false;
+
+    /// <summary>
+    /// Gets or sets a value indicating whether the compilation contained any Slice2 files.
+    /// </summary>
+    public bool ContainsSlice2 { get; set; } = false;
+
+    /// <summary>
+    /// Gets or sets the number of source files in the Slice compilation.
+    /// </summary>
+    public string? SourceFileCount { get; set; } = "";
```

Comment:
Here and for the next property, you should simply keep the default (null).

Also, it's pretty odd for a count to be a string.